https://bugs.launchpad.net/ubuntu/+source/dosfstools/+bug/746262

hack to use just 1 sector for embedded devices.  should be made into a
flag, but this shouldn't break normal systems ...

--- a/src/mkdosfs.c
+++ b/src/mkdosfs.c
@@ -1012,7 +1012,8 @@
 	}
 
 	/* Adjust the reserved number of sectors for alignment */
-	reserved_sectors = align_object(reserved_sectors, bs.cluster_size);
+	/* Wrong rounding for reserved_sectors */
+	/* reserved_sectors = align_object(reserved_sectors, bs.cluster_size); */
 	bs.reserved = CT_LE_W(reserved_sectors);
 
 	/* Adjust the number of root directory entries to help enforce alignment */
